src/components/Footer.js

```diff
@@ -66,11 +66,11 @@ const FooterLink = styled(Link)`
 export default () => (
   <Footer>
     <FooterLink to="/">Home</FooterLink>
-    <FooterLink to="/blog">Blog</FooterLink>
-    <FooterLink to="/about">About</FooterLink>
-    <FooterLink to="/speaking">Speaking</FooterLink>
-    <FooterLink to="/contact">Contact</FooterLink>
-    <FooterLink to="/disclaimer">Disclaimer</FooterLink>
+    <FooterLink to="/blog/">Blog</FooterLink>
+    <FooterLink to="/about/">About</FooterLink>
+    <FooterLink to="/speaking/">Speaking</FooterLink>
+    <FooterLink to="/contact/">Contact</FooterLink>
+    <FooterLink to="/disclaimer/">Disclaimer</FooterLink>
```

Comment:
I miss here a /code/ link

Reply:
Yeah, I just released the code blog and haven't worked out where it fits into the information architecture yet. I think my ultimate goal is to break this theme into a few separate themes (e.g. a blog theme, a speaking engagements theme, a code blog theme) so it's easier for others to use it + a chance for me to work through more complex theme patterns.

Reply:
ok - but just now this blog posts are complete invisible from home page :( 

i just accidentally find it when i see some root.code.lengstorf.com in the old wordpress tutorial :D

Reply:
Yeah, I'm still dark launched while I make sure the old redirects, etc. are working. If you wanted to submit a PR for the footer, that would be super helpful. I'm planning to do more work here in the future; it's on my todo list, but it's not currently a top priority.

Thanks!

Reply:
and one more thought: should a theme be more independent of data (like the links and and more) and these links comes via some properties or graphql queries? i think a little about wordpress themes.

Reply:
Yeah, that's all work that I want to do. This theme is currently far less configurable than I'd like it to be.